diff -Naur scummvm-0.7.1.org/backends/sdl/graphics.cpp scummvm-0.7.1/backends/sdl/graphics.cpp
--- scummvm-0.7.1.org/backends/sdl/graphics.cpp	2005-09-25 22:16:36.030000000 +0200
+++ scummvm-0.7.1/backends/sdl/graphics.cpp	2005-09-25 22:19:20.370000000 +0200
@@ -243,7 +243,8 @@
 	// Create the surface that contains the scaled graphics in 16 bit mode
 	//
 
-	_hwscreen = SDL_SetVideoMode(_screenWidth * _scaleFactor, effectiveScreenHeight(), 16, 
+//	_hwscreen = SDL_SetVideoMode(_screenWidth * _scaleFactor, effectiveScreenHeight(), 16, 
+	_hwscreen = SDL_SetVideoMode(640,480,16,
 		_fullscreen ? (SDL_FULLSCREEN|SDL_SWSURFACE) : SDL_SWSURFACE
 	);
 	if (_hwscreen == NULL) {
diff -Naur scummvm-0.7.1.org/configure scummvm-0.7.1/configure
--- scummvm-0.7.1.org/configure	2005-09-25 22:16:36.100000000 +0200
+++ scummvm-0.7.1/configure	2005-09-25 22:25:18.310000000 +0200
@@ -580,6 +580,16 @@
 	    type_2_byte='short'
 	    type_4_byte='int'
 	    ;;
+	powerpc-tuxbox-linux-gnu)
+	    echo "Cross-compiling to $_host, forcing endianness, alignment and type sizes"
+	    DEFINES="$DEFINES -DUNIX"
+	    _def_endianness='#define SCUMM_BIG_ENDIAN'
+	    _def_align='#define SCUMM_NEED_ALIGNMENT'
+	    _def_powerpc-tuxbox-linux-gnu="#define POWERPC-TUXBOX-LINUX-GNU"
+	    type_1_byte='char'
+	    type_2_byte='short'
+	    type_4_byte='int'
+	    ;;
 	*)
 	    echo "Cross-compiling to unknown target, please add your target to configure."
 	    exit 1
